s3/pkg/service.go

```diff
@@ -138,11 +137,11 @@ func loadGCPDefault(i2e *map[string]*Int2String, e2i *map[string]*String2Int) {
 func loadCephDefault(i2e *map[string]*Int2String, e2i *map[string]*String2Int) {
 	t2n := make(Int2String)
 	t2n[Tier1] = CEPH_STANDARD
-	(*i2e)[OSTYPE_CEPTH] = &t2n
+	(*i2e)[OSTYPE_CEPH] = &t2n
```

Comment:
I remember reviewing a PR for these changes. Was that not merged?

Reply:
I checked the current code of branch master, it still not changed, and I cannot find the PR that have this change.